scripts/version

```diff
@@ -8,17 +8,19 @@ if [ $# -ne 0 ]; then
   echo 'warning: this script does not take any argument.' >&2
 fi
 
+DATE_FMT="%Y-%m-%d"
+SOURCE_DATE_EPOCH="${SOURCE_DATE_EPOCH:-$(date +%s)}"
+date=$(date -u -d "@$SOURCE_DATE_EPOCH" "+$DATE_FMT" 2>/dev/null || date -u -r "$SOURCE_DATE_EPOCH" "+$DATE_FMT" 2>/dev/null || date -u "+$DATE_FMT")
```

Comment:
I agree on `date -u`.

But, is this really necessary ?
`date -u -r "$SOURCE_DATE_EPOCH" "+$DATE_FMT" 2>/dev/null`
In your url I don't see that `SOURCE_DATE_EPOCH` can be file.

Furthermore, `date -u "+$DATE_FMT"` will be never called because of `$(date +%s)` on line 12.